Production/govuk_ios/SwiftUIViews/GroupedList/GroupedList.swift

```diff
@@ -1,22 +1,20 @@
 import SwiftUI
 
 struct GroupedList: View {
-    var content: [GroupedListSection] = []
+    var content: [GroupedListSection]
     var backgroundColor: UIColor?
 
     var body: some View {
-        ZStack {
-            Color(backgroundColor ?? .clear)
-            VStack {
-                ForEach(content, id: \.heading) { section in
-                    GroupedListSectionView(section: section)
+        if content.count >= 1 {
+            ZStack {
+                Color(backgroundColor ?? .clear)
+                VStack {
```

Comment:
Do we need to consider using LazyVStack here?

Reply:
The grouped list section views are already using a lazy stack, im not sure if there side affects of nesting lazy stacks inside one another